app/views/audit.scala.html

```diff
@@ -335,7 +335,9 @@ <h2 id="total-mission-reward" style="display:none"></h2>
                             <p></p>
                         </div>
                         <div style="display:none;">
-                            <img src='@routes.Assets.at("javascripts/SVLabel/img/onboarding/DoubleClick.png")' id="double-click-icon" width="200" alt="Double click icon"/>
+                            <img src='@routes.Assets.at("javascripts/SVLabel/img/onboarding/DoubleClick.png")' id="double-click-icon" width="400" alt="Double click icon"/>
```

Comment:
Ahh right, sorry didn't see this before. Can you add a translation for the alt text here too:
>Icono de doble clic